tests/manage/monitoring/test_workload_fixture.py

```diff
@@ -147,3 +147,17 @@ def test_workload_rbd_cephfs(
     """
     logger.info(workload_storageutilization_50p_rbd)
     logger.info(workload_storageutilization_50p_cephfs)
+
+
+@pytest.mark.libtest
+def test_workload_rbd_cephfs_minimal(
```

Comment:
this is a very good test to be included in one of the testings suites. Can we add it so it will be running in regression cycles?

Reply:
This is a good point, considering we dropped the alerting tests utilizing most of storage capacity out of tier1. I'm going to try to add this into tier1.

Reply:
Implemented in 2c8b10eb42b088fe0f00374f808ed5fd333471ef

The test case just checks whether fio were able to finish the writing job without any failure, logs looks like this:

```
19:06:40 - MainThread - tests.manage.monitoring.test_workload_fixture - INFO - checking fio report results as provided by workload fixtures
19:06:40 - MainThread - tests.manage.monitoring.test_workload_fixture - INFO - starting to check fio run on rbd volume
19:06:40 - MainThread - tests.manage.monitoring.test_workload_fixture - INFO - fio (version fio-3.7) executed simple-write job on rbd volume
19:06:40 - MainThread - tests.manage.monitoring.test_workload_fixture - INFO - starting to check fio run on cephfs volume
19:06:40 - MainThread - tests.manage.monitoring.test_workload_fixture - INFO - fio (version fio-3.7) executed simple-write job on cephfs volume
```